components/Schema.js

```diff
@@ -103,12 +103,20 @@ function SchemaProp({ prop, propName, required = false, path = '', description =
 
 function SchemaPropRow({ prop, propName, required = false, path = '', description = '', isCircular = false }) {
   const acceptedValues = prop.enum() && prop.enum().length ? prop.enum().join(', ') : '_Any_';
-  const types = `${prop.anyOf() ? `anyOf` : ''}${prop.allOf() ? `allOf` : ''}${prop.oneOf() ? `oneOf` : ''}${prop.items() && !Array.isArray(prop.items()) && prop.items().type() ? `${prop.items().type()}` : ''}`;
-  description = `${description || prop.description() || ''}${isCircular ? ' **[CIRCULAR]**': ''}`
+  let type_array = [
+      prop.type() || '',
+      prop.anyOf() ? `anyOf` : '',
+      prop.allOf() ? `allOf` : '',
+      prop.oneOf() ? `oneOf` : '',
+      prop.items() && !Array.isArray(prop.items()) && prop.items().type() ? prop.items().type() : ''
+  ];
+  const types = type_array.filter(type => type != '').join(', ');
```

Comment:
You can use here the guards and simplify the code:

```suggestion
  const types = [
      prop.type(),
      prop.anyOf() && `anyOf`,
      prop.allOf() && `allOf`,
      prop.oneOf() && `oneOf`,
      prop.items() && !Array.isArray(prop.items()) && prop.items().type();
  ].filter(t => t).join(', ');
```